test/e2e/scheduling/predicates.go

```diff
@@ -836,6 +836,7 @@ func initPausePod(f *framework.Framework, conf pausePodConfig) *v1.Pod {
 			Tolerations:                   conf.Tolerations,
 			PriorityClassName:             conf.PriorityClassName,
 			TerminationGracePeriodSeconds: &gracePeriod,
+			NodeName:                      config.NodeName,
```

Comment:
shouldn't this be `conf`?

Reply:
You're right, it should. But I think Wei cleared up my confusion that this approach won't work, so I'm going to close this PR now.
/close